tgui/packages/tgui/interfaces/KitchenMachine.js

```diff
@@ -0,0 +1,103 @@
+import { useBackend } from '../backend';
+import { Button, Section, Table, Flex, Icon, Dimmer } from '../components';
+import { Window } from '../layouts';
+
+export const KitchenMachine = (props, context) => {
+  const { data } = useBackend(context);
+  const { ingredients } = data;
+
+  return (
+    <Window resizable>
+      <Window.Content scrollable display="flex" className="Layout__content--flexColumn">
+        <Operating/>
+        <KitchenTop/>
+        <Section title="Ingredients">
+          <Table className="Ingredient__Table">
+            {ingredients.map((product) => (
+              <Table.Row tr={5} key={product.name}>
+                <tr>
+                  <td>
+                    <Table.Cell bold>{product.name}</Table.Cell>
+                  </td>
+                </tr>
+                <td>
+                  <Table.Cell collapsing textAlign="center">
+                    {product.amount} {product.units}
+                  </Table.Cell>
+                </td>
```

Comment:
Do you need the `<tr>`s and `<td>`s? They seem redundant because they're essentially the same thing as `<Table.Row>` and `<Table.Cell>`.

Reply:
Yeah, weird stuff with CSS and padding makes them required, I tried using padding (and mr and tr and tons of shit) on the cells and rows it didnt do anything

Reply:
Managed to get rid of the <tr>s, removing the tds did some weird stuff because padding and scss